src/app/shared/issue-tables/dropdownfilter.ts

```diff
@@ -1,6 +1,19 @@
 import { Issue } from '../../core/models/issue.model';
 import { Filter } from '../../core/services/filters.service';
 
+type StatusInfo = {
+  type: string;
+  status: string;
+};
+
+/**
+ * Converts a status string into an info object
+ */
+export const statusToType = (statusString: string): StatusInfo => {
+  const [status, type] = statusString.split(' ');
+  return { status, type };
+};
```

Comment:
Is the export necessary?

Reply:
Would it make more sense to name the function something similar to ``infoFromStatus`` since we are returning a type `StatusInfo` object and not just a ``Type`` which the current name seems to imply

Reply:
Fixed!